docs/user-guide/spatial-hashing.ipynb

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
<p>"UXArray" -&gt; "UXarray"</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/1169/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Fixed in https://github.com/UXARRAY/uxarray/pull/1169/commits/fc59591fae435817c73a24ecaf83be0c91ce0998